guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -179,7 +193,7 @@ Create the image with the `hammer compute-resource image create` command. Use th
 ----
 # hammer compute-resource image create --name "Test KVM Image" \
 --operatingsystem "RedHat _version_" --architecture "x86_64" --username root \
---user-data false --uuid "/var/lib/libvirt/images/TestImage.qcow2" \
+--uuid "/var/lib/libvirt/images/TestImage.qcow2" \
```

Comment:
have you tested this? I am just asking because recent tests have not had complaints about this.

Reply:
Good point, I haven't. I am going to revert this one, this can bite. Also `false` does seem logical to me now. If it was set to something else I'd probably dig deeper but this is fine.